src/cueval.cpp

```diff
@@ -157,20 +157,22 @@ RC FC cuEvalR( 		// evaluate pseudocode & return ptr to value
 	rc = cuEval( ip, pmsg, pBadH);	// evaluate to *evStk
 	if (ppv != NULL)				// return pointer to value in eval stack
 		*ppv = evSp;
+#if 0	// TODO64
```

Comment:
Does this need to be resolved before merge?

Reply:
Done